tf2/include/tf2/buffer_core.h

```diff
@@ -46,11 +46,13 @@
 
 #include "LinearMath/Transform.h"
 #include "geometry_msgs/msg/transform_stamped.hpp"
+#include "rcutils/logging_macros.h"
 #include "tf2/buffer_core_interface.h"
 #include "tf2/exceptions.h"
 #include "tf2/transform_storage.h"
 #include "tf2/visibility_control.h"
-#include "rcutils/logging_macros.h"
+
+using std::literals::chrono_literals::operator""ms;
```

Comment:
I'm not a big fan of doing this in a header file, especially.  That means we are importing this anytime someone includes this header file.  I'm going to suggest we just remove this, which solves this problem and also should solve the warning on Windows.

Reply:
This still needs to be defined, just in the .cpp file correct? Also, the warning is just not using an underscore before `ms`, which I'll have fixed later today.

Reply:
> This still needs to be defined, just in the .cpp file correct? Also, the warning is just not using an underscore before `ms`, which I'll have fixed later today.

I mean, we don't *have* to use it; we can always use `std::chrono::millseconds(2500)`, which works just as well.

Reply:
The windows warning should be solved now.